include/netlink/cache.h

```diff
@@ -146,6 +146,10 @@ struct nl_cache_mngr;
 extern int			nl_cache_mngr_alloc(struct nl_sock *,
 						    int, int,
 						    struct nl_cache_mngr **);
+extern int			nl_cache_mngr_alloc_cb(struct nl_sock *,
+						    int, int,
+						    struct nl_cache_mngr **,
+						    struct nl_cb *sync_cb);
```

Comment:
new ABI must be listed in `libnl-3.sym` file.

Reply:
The new entry MUST be added to the bottom of the file. See the comment at the top of the file.